great_expectations/datasource/sqlalchemy_datasource.py

```diff
@@ -239,30 +240,38 @@ def __init__(
             if "engine" in kwargs:
                 self.engine = kwargs.pop("engine")
 
-            # if a connection string or url was provided, use that
-            elif "connection_string" in kwargs:
-                connection_string = kwargs.pop("connection_string")
-                self.engine = create_engine(connection_string, **kwargs)
-                connection = self.engine.connect()
-                connection.close()
-            elif "url" in credentials:
-                url = credentials.pop("url")
-                self.drivername = urlparse(url).scheme
-                self.engine = create_engine(url, **kwargs)
-                connection = self.engine.connect()
-                connection.close()
-
-            # Otherwise, connect using remaining kwargs
             else:
-                (
-                    options,
-                    create_engine_kwargs,
-                    drivername,
-                ) = self._get_sqlalchemy_connection_options(**kwargs)
-                self.drivername = drivername
-                self.engine = create_engine(options, **create_engine_kwargs)
-                connection = self.engine.connect()
-                connection.close()
+                concurrency = (
+                    data_context.concurrency
+                    if data_context is not None
+                    else ConcurrencyConfig()
+                )
+                concurrency.add_sqlalchemy_create_engine_parameters(kwargs)
+
+                # if a connection string or url was provided, use that
```

Comment:
Nit: I've noticed a few one-line comments missing punctuation (here, line 264 below this, and elsewhere in this PR). Can you give this a once over and clean it up? It stuck out to me here because it's worded in such a way that without a period it looks like a sentence got cut off.

Reply:
Done